### ISSUE TITLE
IntelliJ integration and configuration

### ISSUE THREAD

**Issue by sfe-efficy** [vuejs/language-tools#4254]
Hi!

Is any place where I can find documentation about configure the service details with IntelliJ?

I'm following recent changes and I saw there is a new configurable option "hybrid mode". But using the integration in IntelliJ, I don't see clearly how we can configure our services.

With IntelliJ 2024.1 we have this view:

![image](https://github.com/vuejs/language-tools/assets/95854297/8cbfd555-40a3-4836-b18b-63f2aedea074)

Inside configuration we can see:

![image](https://github.com/vuejs/language-tools/assets/95854297/7af040cd-358d-41af-915c-42549c728197)

I don't know if it's possible to provide any kind of argument about where is our configuration, or this services is reading some common places.

In another hand, It was not possible to make working version 2 (latest tested version was 2.0.7, and hybrid mode was not possible to enable in my side).

It wasn't working because was not reporting any kind of error, and application was looking that it was working correctly.

Regards, and thanks for this great work :)

**Comment by so1ve** [comment]
IntelliJ doesn't need hybrid mode. It runs volar as a separated language server, and I'm not sure if they have updated volar to 2.0.

**Comment by loosheng** [comment]
As of 2024.1,IntelliJ hasn't upgraded the v2 version of volar yet, they're currently under observation, so you can pay attention to this issue：[WEB-65823](https://youtrack.jetbrains.com/issue/WEB-65823/update-to-vue-language-server-v2)

**Comment by sfe-efficy** [comment]
Yes, I know, but It's possible to use another version.

![image](https://github.com/vuejs/language-tools/assets/95854297/59f73ee9-ed18-4b3a-bf77-580451610364)

When I use the latest version (like 2.0.12), IntelliJ stop reporting errors, but this is happen too using **vue-tsc** too:

`vue-tsc --noEmit --project crm`

```
> webapp@0.1.0 postbuild:crm
> vue-tsc --noEmit --project crm


Process finished with exit code 0
```

But, I know that I have errors in code:

![image](https://github.com/vuejs/language-tools/assets/95854297/ab059058-86fa-4ec4-952a-f77c3001614a)


In any case, my question was more related about how the services can be configured? Because, normally when I saw this topic all details to configure this services is focus on VS.

**Comment by sfe-efficy** [comment]
Hi again, some updates.

I retested again with latest version, 2.0.6 and I was having same problem.

But, this time I executed "vue-tsc" command with "--watch" mode and I saw an error about "ScriptKind" definition not found. It's an enum defined directly in "typescript".

I checked my "typescript" version and it was 5.1.6.
It suppose that this enum is defined in this version https://raw.githubusercontent.com/microsoft/TypeScript/v5.1.6/lib/typescript.d.ts

In any case, I tried to update typescript to 5.4.5 and now works fine :)

Before the services was not working, it was not listed in IntelliJ.

![image](https://github.com/vuejs/language-tools/assets/95854297/baadb3e6-cf2b-47cd-be2c-6ce214b79224)

So, maybe it's missing a minimal required typescript version for this library.
